common/services/prismic/transformers/collection-venues.ts

```diff
@@ -92,3 +92,17 @@ export function transformCollectionVenues(
     return Number(a.order) - Number(b.order);
   });
 }
+
+// venue is passed down as JSON, so need to convert the date strings back to Moment objects
+export function convertTimeStringsBackToMoments(venue: Venue): Venue {
+  return {
+    ...venue,
+    openingHours: {
+      regular: venue.openingHours.regular,
```

Comment:
minor: let's unpack so we don't lose any new fields we add to this object.

```suggestion
      ...venue.openingHours,
```